src/app/bisoncoin/src/pages/LoginPage.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="home">
+    <img
+      width="300px"
+      height="300px"
+      alt="BisonCoin logo"
+      src="../assets/BisonCoin.png"
+    />
+    <TextInput label="Username" />
+    <TextInput label="Password" />
+    <Button label="Login" />
+    <a href="hello">Don't have an account? Create one now!</a>
```

Comment:
Updated HREF to say TODO instead. Additionally, move this hardcoded text into a separate file as mentioned prior